circle.yml

```diff
@@ -65,6 +65,7 @@ jobs:
                       echo "Version mismatch, v${NODE_VERSION} != ${ACTUAL_VERSION}"
                       exit 1;
                     fi
+                    echo "Version ${ACTUAL_VERSION} confirmed"
```

Comment:
I added this so that I can see the current version echoed out in CI when successful